src/mlpack/methods/ann/init_rules/lecun_normal_init.hpp

```diff
@@ -0,0 +1,110 @@
+/**
+ * @file lecun_normal_init.hpp
+ * @author Dakshit Agrawal and Prabhat Sharma
+ *
+ * Intialization rule given by Lecun et. al. for neural networks and
+ * also mentioned in Self Normalizing Networks.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_ANN_INIT_RULES_LECUN_NORMAL_INIT_HPP
+#define MLPACK_METHODS_ANN_INIT_RULES_LECUN_NORMAL_INIT_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <mlpack/core/math/random.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * This class is used to initialize weight matrix with the Lecun Normalization
+ * initialization rule.
+ *
+ * For more information, the following papers can be referred to:
+ *
+ * @code
+ * @inproceedings{conf/nips/KlambauerUMH17,
```

Comment:
Do you mind to reformat the citation, here is an example: 
https://github.com/mlpack/mlpack/blob/b6b96de5df581f9cf7d4d0db1015d7fbb685fee4/src/mlpack/core/optimizers/adam/adam_update.hpp#L32-L40